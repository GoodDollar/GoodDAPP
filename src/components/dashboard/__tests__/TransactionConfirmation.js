import React from 'react'

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'
import { getWebRouterComponentWithMocks } from './__util__'

describe('TransactionConfirmation', () => {
  it('renders without errors', () => {
    const TransactionConfirmation = getWebRouterComponentWithMocks('../TransactionConfirmation')
    const tree = renderer.create(<TransactionConfirmation />)
    expect(tree.toJSON()).toBeTruthy()
  })

  it('matches snapshot', () => {
    const TransactionConfirmation = getWebRouterComponentWithMocks('../TransactionConfirmation')
    const component = renderer.create(<TransactionConfirmation />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
