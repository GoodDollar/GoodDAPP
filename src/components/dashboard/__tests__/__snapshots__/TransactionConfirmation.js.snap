// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TransactionConfirmation matches snapshot 1`] = `
<div
  className="css-view-1dbjc4n r-flex-13awgt0 r-pointerEvents-12vffkv"
>
  <div
    className="css-view-1dbjc4n r-display-6koalj r-flexDirection-eqz5dr r-flexGrow-16y2uox r-maxHeight-87ditf r-overflowX-lltvgl r-overflowY-1rnoaur r-padding-edyy15 r-position-bnwqim r-width-13qz1uu"
    data-name="viewWrapper"
    style={
      Object {
        "backgroundImage": "linear-gradient(3.141592653589793rad,#00AFFF,#2DC0F7,#28C0EF,#23C0E7,#1EC1DF,#19C1D7,#14C1CF,#0FC2C7,#0FC2C7,#0AC2BF,#05C2B7,#00C3AF)",
      }
    }
  >
    <div
      className="css-view-1dbjc4n r-alignItems-1awozwy r-backgroundColor-14lw9ot r-borderRadius-1f0042m r-display-6koalj r-flexDirection-eqz5dr r-flexGrow-16y2uox r-justifyContent-1wtj0ep r-paddingHorizontal-1e081e0 r-paddingVertical-1yzf0co"
    >
      <div
        className="css-view-1dbjc4n r-alignItems-1awozwy r-flexDirection-eqz5dr r-justifyContent-1wtj0ep r-marginTop-1t01tom"
      >
        <div
          className="css-text-901oao r-alignItems-1awozwy r-color-cddv6f r-display-6koalj r-fontFamily-vnw8o6 r-fontSize-evnaw r-fontWeight-b88u0q r-justifyContent-1777fci r-lineHeight-17rnw9f r-marginBottom-1ifxtd0 r-textAlign-q4m81j r-textDecorationLine-13wfysu r-textTransform-3twk1y"
          dir="auto"
        >
          Complete Your Transaction:
        </div>
        <div
          className="css-view-1dbjc4n r-alignItems-1oszu61 r-display-6koalj r-flexDirection-eqz5dr r-justifyContent-1h0z5md r-marginHorizontal-1jgb5lz r-textAlign-fdjqy7"
        >
          <div
            className="css-text-901oao r-color-cddv6f r-fontFamily-vnw8o6 r-fontSize-ubezar r-fontWeight-majxgm r-letterSpacing-uz4sb0 r-lineHeight-vrz42v r-textAlign-fdjqy7 r-textDecorationLine-13wfysu r-textTransform-3twk1y"
            dir="auto"
            letterSpacing={0.16}
          >
            <span
              className="css-text-901oao css-textHasAncestor-16my406 r-color-cddv6f r-fontFamily-vnw8o6 r-fontSize-ubezar r-fontWeight-b88u0q r-letterSpacing-uz4sb0 r-lineHeight-vrz42v r-textAlign-fdjqy7 r-textDecorationLine-13wfysu r-textTransform-3twk1y"
              dir="auto"
              letterSpacing={0.16}
            >
              1. 
            </span>
            Copy the link below
          </div>
          <div
            className="css-text-901oao r-color-cddv6f r-fontFamily-vnw8o6 r-fontSize-ubezar r-fontWeight-majxgm r-letterSpacing-uz4sb0 r-lineHeight-vrz42v r-textAlign-fdjqy7 r-textDecorationLine-13wfysu r-textTransform-3twk1y"
            dir="auto"
            letterSpacing={0.16}
          >
            <span
              className="css-text-901oao css-textHasAncestor-16my406 r-color-cddv6f r-fontFamily-vnw8o6 r-fontSize-ubezar r-fontWeight-b88u0q r-letterSpacing-uz4sb0 r-lineHeight-vrz42v r-textAlign-fdjqy7 r-textDecorationLine-13wfysu r-textTransform-3twk1y"
              dir="auto"
              letterSpacing={0.16}
            >
              2. 
            </span>
            Share it with sender
          </div>
          <div
            className="css-text-901oao r-color-cddv6f r-fontFamily-vnw8o6 r-fontSize-ubezar r-fontWeight-majxgm r-letterSpacing-uz4sb0 r-lineHeight-vrz42v r-textAlign-fdjqy7 r-textDecorationLine-13wfysu r-textTransform-3twk1y"
            dir="auto"
            letterSpacing={0.16}
          >
            <span
              className="css-text-901oao css-textHasAncestor-16my406 r-color-cddv6f r-fontFamily-vnw8o6 r-fontSize-ubezar r-fontWeight-b88u0q r-letterSpacing-uz4sb0 r-lineHeight-vrz42v r-textAlign-fdjqy7 r-textDecorationLine-13wfysu r-textTransform-3twk1y"
              dir="auto"
              letterSpacing={0.16}
            >
              3. 
            </span>
            Sender approves request
          </div>
          <div
            className="css-text-901oao r-color-cddv6f r-fontFamily-vnw8o6 r-fontSize-ubezar r-fontWeight-majxgm r-letterSpacing-uz4sb0 r-lineHeight-vrz42v r-textAlign-fdjqy7 r-textDecorationLine-13wfysu r-textTransform-3twk1y"
            dir="auto"
            letterSpacing={0.16}
          >
            <span
              className="css-text-901oao css-textHasAncestor-16my406 r-color-cddv6f r-fontFamily-vnw8o6 r-fontSize-ubezar r-fontWeight-b88u0q r-letterSpacing-uz4sb0 r-lineHeight-vrz42v r-textAlign-fdjqy7 r-textDecorationLine-13wfysu r-textTransform-3twk1y"
              dir="auto"
              letterSpacing={0.16}
            >
              4. 
            </span>
            GoodDollars are transferred
          </div>
        </div>
      </div>
      <div
        className="css-view-1dbjc4n r-alignItems-1awozwy r-height-1x8dyos r-marginTop-1t01tom r-width-13qz1uu"
      >
        <confirmTransaction.svg />
      </div>
      <div
        className="css-view-1dbjc4n r-marginTop-1t01tom"
      >
        <div
          className="css-view-1dbjc4n r-cursor-1loqt21 r-touchAction-1otgn73 r-transitionDuration-eafdt9 r-transitionProperty-1i6wzkk r-userSelect-lrvibr r-width-13qz1uu"
          data-focusable={true}
          disabled={false}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          tabIndex="0"
        >
          <div
            className="css-view-1dbjc4n"
            style={
              Object {
                "width": "100%",
              }
            }
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
